### PR TITLE
General styling

### PR DESCRIPTION
**Pull request by zelenovakat**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -18,26 +18,26 @@ const App = () => {
         <Router>
           <div>
             <Switch>
-              <Route path="/trips/:tripId/AddEvent">
+              <Route path="/trips/:tripId/new-trip">
                 <List>
                   <Link to="/trips">
-                    <FontAwesomeIcon icon={faAngleLeft} /> trips
+                    <FontAwesomeIcon icon={faAngleLeft} /> Back
                   </Link>
```
Comment (igkra) [inline src/App.js:25]: You have this section repeated in 3 places.
Maybe it is possible to move it up to the row 20 before `<Switch>` ?

File: src/Pages/NewTrip.jsx [hunk 2]
```diff
@@ -17,23 +19,25 @@ export default function NewTrip() {
 
   return (
     <Form onSubmit={handleSubmit(onSubmit)} autoComplete="off">
-      <Lable>name</Lable>
+      <Title>Create A New Trip</Title>
+
+      <StyledLable>Name</StyledLable>
```
Comment (igkra) [inline src/Pages/NewTrip.jsx:24]: typo. you probably meant `<StyledLabel>`

the same below

File: src/Pages/NewTrip.jsx [hunk 3]
```diff
@@ -42,40 +46,41 @@ const Form = styled.form`
...
+  outline: none;
+
+  &:focus {
+    border: 2px solid #4687a8;
+  }
+`
+const StyledLable = styled.label`
+  margin-left: 10px;
+  color: #484848;
+  font-weight: normal;
```
Comment (igkra) [inline src/Pages/NewTrip.jsx:77]: isn't `font-weight` normal by default?
you may remove this line.

File: src/App.js (hunk 1 above)
Comment (zelenovakat) [reply src/App.js:25]: i didn't know how i can remove it for general page, so i left it as it is

**Pull request merged**